extensions/interactions/base.py

```diff
@@ -83,6 +83,8 @@ class BaseInteraction(object):
     is_linear = False
     # Whether this interaction supports training and fuzzy classification.
     is_trainable = False
+    # Whether this interaction supports the string classifier.
+    is_string_classifier_trainable = False
```

Comment:
Please put a TODO in here to remove this once classifier_services is generalized. This is probably not a pattern we want to encourage elsewhere, but it seems necessary right now.